src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
```

Comment:
Let's use DB/schema names which are suitable for the specific domain. (in this case it's better to use some name related to books)